### DISCUSSION TITLE
ip address limitations

### DISCUSSION THREAD

**Discussion by sw41** [blakeblackshear/frigate#12148] [General]
I have frigate 0.13.2 in HA 2024.6.1 with go2rtc 1.8.4 at 10.241.141.33 and it works great. Cam 1 is at 10.241.141.67 and streams.  When I change the IP to cam 2 at 10.241.135.27, I get the 404 error and it cannot see the stream.  I can see the stream with the cam's app not in HA.  

Are we limited to /24??

**Comment by sw41** [comment]
guessing this is a yes, that frigate only works in a /24 network??

**Comment by blakeblackshear** [comment]
There is nothing I am aware of that would do anything to limit the network in Frigate.

**Reply by sw41** [reply]
Thanks!
